src/components/ProgressIndicator/ProgressIndicator.md

```diff
@@ -0,0 +1,40 @@
+# Progress Indicator
+Displays the progress of an operation when the percentage complete is known. Unable to determine how much of the operation is complete? Use a Spinner instead.
```

Comment:
I would also say that this is the "determinate" progress indicator (in contrast to the Spinner being indeterminate).